frontend/pages/beta/model/new.tsx

```diff
@@ -50,6 +52,7 @@ export default function NewModel() {
 
     if (!response.ok) {
       const error = await getErrorMessage(response)
+      setLoading(false)
```

Comment:
This should be moved outside of this `if` statement. We always want to `setLoading(false) when we get a response.